ctd/utilities.py

```diff
@@ -66,13 +66,11 @@ def ufunclike(xs):
 
     return ufunclike
 
-
 def normalize_names(name):
     name = name.strip()
     name = name.strip('*')
     return name
 
-
```

Comment:
Try to avoid non-code changes. It makes review and tracking significant changes much easier.
(Or, we can send a PR only to change style.)